aeron-client/src/main/java/io/aeron/ChannelUriStringBuilder.java

```diff
@@ -2171,6 +2173,56 @@ public ChannelUriStringBuilder maxResend(final Integer maxResend)
         return this;
     }
 
+    /**
+     * The stream id of the channel.
+     *
+     * @return the stream or null of no streamId is set.
+     */
+    public Integer streamId()
+    {
+        return streamId;
+    }
+
+    /**
+     * The stream id of the channel.
+     *
+     * @param streamId the stream id.
+     * @return this for a fluent API.
+     */
+    public ChannelUriStringBuilder streamId(final Integer streamId)
+    {
+        this.streamId = streamId;
+        return this;
+    }
```

Comment:
Please could add tests to ChannelUriStringBuilderTest for coverage.  Adding the additional fields to `shouldBuildChannelBuilderUsingExistingStringWithAllTheFields` would also be useful.

Reply:
@mikeb01 This is already implemented on a master branch.